MonoGame.Framework/Content/ContentManager.cs

```diff
@@ -506,6 +540,27 @@ private ContentReader GetContentReaderFromXnb(string originalAssetName, Stream s
 
             return reader;
         }
+    
+        Texture2D LoadTexture2DHeuristically(string assetName)
```

Comment:
Any chance we can change the name of this method to something a little more straight forward?
`LoadTexture2DFromImageFile` or something like that? 

Its a small thing, however having simple method names makes things easier to maintain IMHO.

Reply:
Thanks!